packages/core/src/errors.ts

```diff
@@ -94,6 +94,7 @@ export const enum RuntimeErrorCode {
   TYPE_IS_NOT_STANDALONE = 907,
   MISSING_ZONEJS = 908,
   UNEXPECTED_ZONE_STATE = 909,
+  UNSUPPORTED_STANDARD_DECORATORS = 910,
```

Comment:
This code would overlap with the `UNSAFE_IFRAME_ATTRS` one below (the `-` sign means that the error code has a guide).
```suggestion
  UNSUPPORTED_STANDARD_DECORATORS = 911,
```